huaweicloud/services/dataarts/resource_huaweicloud_dataarts_dataservice_app.go

```diff
@@ -73,7 +73,6 @@ func ResourceDataServiceApp() *schema.Resource {
 			"app_type": {
 				Type:        schema.TypeString,
 				Optional:    true,
-				Computed:    true,
```

Comment:
Risk of incompatible changes.
Keep the attribute `Computed`.

Reply:
done.